Gemfile

```diff
@@ -73,7 +73,7 @@ gem 'stripe', '~> 5.28.0' # we need the stripe gem because activemerchant can no
 gem 'acts_as_list', '~> 0.9.17'
 gem 'braintree', '~> 2.93'
 gem 'bugsnag', '~> 6.26'
-gem 'cancancan', '~> 3.0.0'
+gem 'cancancan', '~> 3.6.0'
```

Comment:
`3.1.0` was the first version to fix keyword arguments: https://github.com/CanCanCommunity/cancancan/blob/develop/CHANGELOG.md#310

But going up to `3.6.0` seemed OK.